enterprise/views.py

```diff
@@ -499,7 +503,23 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find course details for course ID: %s', course_id)
             raise Http404
 
-        platform_name = configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
+        enterprise_customer = get_enterprise_customer_or_404(enterprise_uuid)
+
+        enrollment_client = EnrollmentApiClient()
+        enrolled_courses = enrollment_client.get_enrolled_courses(request.user.username)
```

Comment:
Good idea - I've added this endpoint to the EnrollmentApiClient and we're now using that instead.